add-on/panels/js/.eslintrc

```diff
@@ -1,13 +1,12 @@
 {
-  "ecmaFeatures": {
-   // These are on for this directory for .jsm and content/js files.
-    "blockBindings": true,
-    "arrowFunctions": true,
-    "destructuring": true,
-    "generators": true,
-    "spread": true,
-    "restParams": true,
-    "objectLiteralShorthandMethods": true
+  "env": {
+    "es6": true,
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true,
+    },
+    "ecmaVersion": 6,
   },
   "rules": {
     "generator-star-spacing": [2, "after"]
```

Comment:
We can simplify some of these `.eslintrc` files by deleting/moving this file to `add-on/.eslintrc` with just:
```json
{
  "env": {
    "es6": true,
  },
  "parserOptions": {
    "ecmaVersion": 6,
  },
}
```
And removing `env` and `parserOptions` (and the `generator-star-spacing` rule) from the descendant `.eslintrc`s.